src/const_openevse.js

```diff
@@ -0,0 +1,303 @@
+        // toolbar_left                     // Mapping of states where each state has a list of entity items which will appear for corresponding state at left side of toolbar at bottom of card
+        // toolbar_right                    // Mapping of states where each state has a list of entity items which will appear for corresponding state at right side of toolbar at bottom of card
+
+    // ENTITY ITEMS (APPLY THE ONES YOU NEED)
+        //     entity_id: '',                  // entity id
+        //     attribute: '',                  // attribute is used as value if specified
+        //     unit: '',                       // unit if you want to override entity unit
+        //     unit_show: true,                // show unit next to value
+        //     unit_showontext: true,          // show unit next to value in tooltip text
+        //     text: '',                       // text to be used instead of entity friendly-name (do not use dots '.' and apply translation key to achieve translation)
+        //     service: '',                    // service on format 'domain.service'
+        //     service_data: {'test','test'},  // service data for the service call
+        //     icon: '',                       // icon to be used instead of entity icon
+        //     round: 0,                       // round to specified number of decimals (integer)
+        //     type: '',                       // type
+        //     calc_function: ''               // define entity_id as 'calculated' and specify min,max,mean,sum here to calculate
+        //     calc_entities: ''               // entities to calculate from above feature
+        //     conditional_entity: ''          // if you want the entity_id to be shown conditionally, specify a on/off or true/false sensor here
+        //     conditional_attribute: ''       // if you prefer the conditional showing of entity to be based on an attribute, define it here
+        //     conditional_invert: ''          // if you prefer to invert the conditional showing of an entity to show when false, invert by true
+
+    // SPECIAL TOKENS
+        // #ENTITYPREFIX#                      // This will be replaced with what is found from main sensor entity after removing MAIN_ENTITY_BASE
+        // #SERVICEID#                         // A replacement used in the service call, typically for a chargerid or something that must be part of the data when calling service of a specific charger.
+        // #SERVICEVAL#                        // A replacement used in the service call, typically for the value from a dropdown or similar. Use this in the template where for instance a current limit is supposed to be sent to a charger.
+
+
+    //NAME, LOCATION, STATUS ETC
+    name: 'Charger',
+    location: 'Home',
+    status: {
+        entity_id: 'sensor.#ENTITYPREFIX#_charging_status',
+    },
+
+    // OVERRIDE CURRENTLIMITS
+    currentlimits: [0, 6, 10, 16, 20, 25, 32],
```

Comment:
Would not adding 12 and 48 (max that OpenEVSE supports) be a good idea here?
12A would be the limit you'd want on a 15A breaker, while 48A would be idea for a 50-60A breaker.

Reply:
Hi @firstof9 - openevse is just work-in-progress right now, so perhaps it could be a good idea. Anyway, the way it is now anyone can customize these values to fit their need. I wouldn't agree that 12A would fit 15A breaker though, but I've seen the discussions on the big o' internet ;) if the dropdown menu becomes too big it is unpractical I've seen, but perhaps I could do something about the formatting of it.

Reply:
Ah so these are just the defaults and you can override them with config?

Reply:
Yes, fully customizable. But plan is to make "templates" that matches normal setup for most people, to make it easier.

I know about the 80% rule of thumb, but that's what it is too :)